man/kernel-command-line.xml

```diff
@@ -97,6 +97,18 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>systemd.boot_breakpoint=</varname></term>
+        <term><varname>rd.systemd.boot_breakpoint=</varname></term>
```

Comment:
I wonder if we should use similar syntax to dracut for familiarity. e.g. `systemd.break=`

Reply:
> I wonder if we should use similar syntax to dracut for familiarity. e.g. `systemd.break=`

For me your suggestion sounds good, and it's easier to type it at boot. Let's get more feedback.

Reply:
> In that case we'd have to generalize to "breakpoint" instead of boot breakpoint.

Therefore, I also renamed the kernel cmdline option.